inference-engine/thirdparty/clDNN/src/gather_nd.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+// Copyright (c) 2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+#include "gather_nd_inst.h"
+
+#include "primitive_type_base.h"
+#include "error_handler.h"
+#include "json_object.h"
+#include <string>
+
+namespace cldnn {
+primitive_type_id gather_nd::type_id() {
+    static primitive_type_base<gather_nd> instance;
+    return &instance;
+}
+
+layout gather_nd_inst::calc_output_layout(gather_nd_node const& node) {
+    auto op = node.get_primitive();
+
+    auto input_layout_orgin = node.input(0).get_output_layout();
```

Comment:
I amn't sure but maybe "input_layout_origin"?

Reply:
Ooops. It was a typo. I've fixed.